uniffi_bindgen/src/bindings/python/templates/CallableBody.py

```diff
@@ -29,7 +29,7 @@
 
 {%- match callable.throws_type.ty %}
 {%- when Some(e) %}
-{%-    match e.ty %}
+{%-    match e.ty.canonical() %}
```

Comment:
I think the right thing here is a new pipeline pass, eg, "error.rs"? It would walk `Callable` and adjust `throws_type` directly. This would also then handle CallbackInterfaceImpl.py